homeassistant/components/mazda/sensor.py

```diff
@@ -163,6 +211,22 @@ def _rear_right_tire_pressure_value(data, unit_system):
         is_supported=_rear_right_tire_pressure_supported,
         value=_rear_right_tire_pressure_value,
     ),
+    MazdaSensorEntityDescription(
```

Comment:
Should any of these get a `state_class`? 

https://developers.home-assistant.io/blog/2021/08/16/state_class_total?_highlight=state&_highlight=class#state-classes

Reply:
@bdraco  Good idea! I was going to do this in a separate PR, but since you asked about it, I went ahead and added state classes to this PR.